docs/components/AnimatedPresence/Web.stories.tsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState } from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import {
+  AnimatedPresence,
+  AnimatedPresenceTransitions,
+} from "@jobber/components/AnimatedPresence";
+import { Button } from "@jobber/components/Button";
+import { Text } from "@jobber/components/Text";
+import { Card } from "@jobber/components/Card";
+import { Content } from "@jobber/components/Content";
+import { Flex } from "@jobber/components/Flex";
+import { Divider } from "@jobber/components/Divider";
+import { Heading } from "@jobber/components/Heading";
+import { AnimatedSwitcher } from "@jobber/components/AnimatedSwitcher";
+
+export default {
+  title: "Components/Utilities/AnimatedPresence/Web",
+  component: AnimatedPresence,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof AnimatedPresence>;
+
+const BasicTemplate: ComponentStory<typeof AnimatedPresence> = args => {
```

Comment:
Overall this looks great, I have very little feedback. That said, there's something in the storybook stories that I want to call out. It might only exist there, but I'm not 100% sure. Might in the real code too.

So it looks like on the Basic story (might exist elsewhere too) there's a margin separating each item. And it looks like that margin is not part of the animation.

This is a nitpick, but I've spent so many years looking at animations that it jumped out to me.

Because the margin is not animated, there appears to be a bit of a 'jump' at the end of the transition. I took a video of it in motion and broke it down to two frames to show the jump happening. The first frame is two frames back from the jump, where you can see sort of see the text as it's taken out via opacity/height. Then the second frame shows right before the margin is removed (the div is taken out of the DOM) and then the final frame where it's out.

We can chat on a call if you want to see it better. :)

![image](https://github.com/GetJobber/atlantis/assets/145565743/1a8cd21b-c1d6-4863-9dfb-198ec1d86c9e)
![image](https://github.com/GetJobber/atlantis/assets/145565743/27f27453-4a02-42fe-b689-10ba48b9f87a)
![image](https://github.com/GetJobber/atlantis/assets/145565743/e627461d-1147-4f38-ba41-7e96f9ed2833)

Reply:
yeah, noticed that too. Unfortunately, the solution for that might just be allowing `AnimatedPresence` to set a spacing instead of a wrapping `Content` since framer needs to know about those values

Reply:
ok, latest version of framer motion actually fixes that and we're due for an update anyway so I'll do that first haha

Reply:
Fixed now!